cover_agent/AICaller.py

```diff
@@ -23,7 +25,12 @@ def call_model(self, prompt, max_tokens=4096):
             tuple: A tuple containing the response generated by the language model, the number of tokens used from the prompt, and the total number of tokens in the response.
         """
         messages = [{"role": "user", "content": prompt}]
-        response = litellm.completion(model=self.model, messages=messages, max_tokens=max_tokens, stream=True)
+
+        # API base exception for Ollama and Hugging Face models
```

Comment:
@EmbeddedDevops1 
The 'if' is quite ugly. 
i don't want to create gaps so I will merge, but please fix it tomorrow

that's exactly the things we solved at 
https://github.com/Codium-ai/cover-agent/pull/20/files

![image](https://github.com/Codium-ai/cover-agent/assets/21198860/4725f8f4-c129-462b-bb07-83d8ce8e10a1)